conman/routes/tests/test_handlers.py

```diff
@@ -101,12 +100,84 @@ def test_handle_no_url_match(self):
         self.assertFalse(dummy_view.called)
 
 
+class URLConfHandlerCheckTest(TestCase):
+    """Tests for URLConfHandler.check()."""
+    def test_no_urlconf(self):
+        """When the route has no urlconf, return an error."""
+        class RouteWithNoURLConf(Route):
+            handler_class = URLConfHandler
+
+        errors = URLConfHandler.check(RouteWithNoURLConf)
+        expected = Error(
+            'RouteWithNoURLConf must have a `urlconf` attribute.',
+            hint=(
+                'The urlconf must be a dotted path. ' +
+                'This is a requirement of URLConfHandler.'
+            ),
+            obj=RouteWithNoURLConf,
+        )
+        self.assertEqual(errors, [expected])
+
+    def test_static_view(self):
+        """When the Route has a staticmethod view, all's well."""
+        class RouteWithURLConf(Route):
+            handler_class = URLConfHandler
+            urlconf = 'a.dotted.path'
+
+        errors = URLConfHandler.check(RouteWithURLConf)
+        self.assertEqual(errors, [])
```

Comment:
This test doesn't appear to have a view, but the test name and docstring refer to one.